### DISCUSSION TITLE
Is `update` supposed to change enum values when they're omitted from the call?

### DISCUSSION THREAD

**Discussion by shohoney** [prisma/prisma#16107] [Q&A] [answered]
We have a model that has an enum as one of its columns.  If we call update with a partial of the model, that excludes the enum, we're observing that the enum is being changed to the `0` value of the enum. I'd expect the omission of the value to not update the existing value that's set.  Is this understanding incorrect? We're using Postgres.  Happy to provide any additional information that'd help explain this better. Our version of prisma is 3.13.0

**Comment by shohoney** [comment, marked as answer]
We isolated the mistake on our end.  Please ignore!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
